src/main/java/org/solid/testharness/http/SolidClient.java

```diff
@@ -30,10 +32,15 @@
     private Client client;
 
     public SolidClient() {
-        client = ClientRegistry.getClient(ClientRegistry.DEFAULT);
+        final ClientRegistry clientRegistry = CDI.current().select(ClientRegistry.class).get();
+        client = clientRegistry.getClient(ClientRegistry.DEFAULT);
     }
     public SolidClient(final String user) {
-        client = ClientRegistry.getClient(user);
+        final ClientRegistry clientRegistry = CDI.current().select(ClientRegistry.class).get();
+        client = clientRegistry.getClient(user);
+        if (client == null) {
+            throw new TestHarnessInitializationException("Client '%s' has not been registered yet", user);
+        }
```

Comment:
I don't understand what role this plays with respect to the goal of the PR :-)

Reply:
It is something that was sensible to do even if not part of the PR, however it was essential to the tests which were within the intent of the PR. It allows me to mock clients that are used in tests.

Reply:
OK, sounds good.